bin/models.py

```diff
@@ -30,11 +30,11 @@ def get_by_id(cls, snippet_id):
 
         code = snippet[b'code'].decode('utf-8')
         views_left = int(snippet[b'views_left'].decode('utf-8'))
-        if views_left == -1:
+        if views_left == 0:
             pass
         elif views_left == 1:
             database.delete(snippet_id)
         else:
-            database.hincrby(snippet_id, 'views_left', -1)
+            database.hincrby(snippet_id, 'views_left', 0)
```

Comment:
```suggestion
            database.hincrby(snippet_id, 'views_left', -1)
```
Ça il faut laisser comme c'était, ça sert à decrémenter.